codes/classical/properties/locally_recoverable.yml

```diff
@@ -0,0 +1,39 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: locally_recoverable
+
+
+name: 'Locally Recoverable code'
+
+description: 'A code \(C\) over \(\mathbb{F}_q^{n}\) with length \(n\) is \(r\)-locally recoverable, or have locality \(r\), if \(\forall i \in [n]\), \(\exists I_i \subset [n]\setminus i\)~ s.t. \(|I_i|\leq r\) and the projection of the set \(\mathcal{C}(i,a)=\{x\in C : x_i=a\}\) on to the coordinates in \(I_i\), i.e., \(\mathcal{C}_{I_i}(i,a)\) is disjoint from \(\mathcal{C}_{I_i}(i,a^\prime)\) where \(a\neq a^\prime\). In other words, given a codeword \(x\) and coordinate \(i\), \(x_i\) can be recovered from at most \(r\) other coordinates of \(x\). An \(r\)-locally recoverable code with length \(n\) and dimension \(k\) is denoted as (n,k,r) LRC code.'
+
+features:
+    rate: 'The rate of (n,k,r) LRC codes satisfies
+            \begin{align}
+                \frac{k}{n}\leq\frac{r}{r+1}
+            \end{align}'
+      
+
+notes:
+  - '\(r\)-locally recoverable codes are not necessarily linear.'
+  - 'The distance, \(d\), of (n,k,r) LRC codes satisfies
+        \begin{align}
+            d\leq n-k-\left \lceil\frac{k}{r}\right \rceil+2
+        \end{align}'
+  - 'It should be obvious from definition that \(r\leq k\). Moreover for \(k=r\), the bound on distance gives Singleton bound.'
+  - 'Generalized Singleton bound does not account for \(q\)-ary alphabet size. A more generalized bound(including non-linear case) is given in \cite{manual:{V. R. Cadambe and A. Mazumdar, "Bounds on the Size of Locally Recoverable Codes," in IEEE Transactions on Information Theory, vol. 61, no. 11, pp. 5787-5794, Nov. 2015, doi: 10.1109/TIT.2015.2477406.}}'
+  - 'A study on parameters of LRC(t-LRC) codes together with known bounds can be found in \cite{manual:{I. Tamo, A. Barg and A. Frolov, "Bounds on the Parameters of Locally Recoverable Codes," in IEEE Transactions on Information Theory, vol. 62, no. 6, pp. 3070-3083, June 2016, doi: 10.1109/TIT.2016.2518663.}}'
+
+relations:
+  parents:
+    - code_id: ecc
+      detail: ''
+  cousins:
```

Comment:
Regenerating codes

Reply:
FYI: I added it as a commented out line, since someone else is already submitting Regenerating codes. for Codes with availability, Codes for distributed storage, I added them as new files(empty) (both seems to be property codes) to property folder. Also I added Codes for distributed storage as a parent to reed solomon as well(is that ok?)

Reply:
I've created a stub entry in the folder classical/properties/distributed_storage. For now, I'm following FIg. 31.12 of Huffman Kim Sole, which puts availability as a child of this. Keep the parents/cousins as in the stub, and add the rest next committ.